tracer/src/Datadog.Trace/Util/DbCommandCache.cs

```diff
@@ -39,7 +39,13 @@ internal static class DbCommandCache
 
         public static KeyValuePair<string, string>[] GetTagsFromDbCommand(IDbCommand command)
         {
-            var connectionString = command.Connection.ConnectionString;
+            var connectionString = command.Connection?.ConnectionString;
```

Comment:
This was throwing NREs in unit tests and silently caught. Probably never happens in real apps but better to be save?